owncloudApp/src/main/res/values/strings.xml

```diff
@@ -765,6 +765,8 @@
     <string name="release_notes_4_3_0_subtitle_show_app_provider_icon_from_endpoint">More appropriate icons have been added to the "Open in (web)" option on the operations menu</string>
     <string name="release_notes_4_3_0_title_av_offline_files_removed_locally_with_local_only_option">Displayed \"Local only\" option properly and prevented deletion of available offline files in the remove dialog</string>
     <string name="release_notes_4_3_0_subtitle_av_offline_files_removed_locally_with_local_only_option">\"Local only\" option in remove dialog will be shown when the selected folder contains at least one downloaded file, ignoring those available offline. If the "Local only" option is displayed and clicked, available offline files will not be deleted</string>
+    <string name="release_notes_4_3_0_title_show_same_remove_dialog_for_every_file_folder">Improved thumbnails in the delete files dialog</string>
+    <string name="release_notes_4_3_0_subtitle_show_same_remove_dialog_for_every_file_folder">Thumbnails have been added in remove dialog for all files and not only for images and .txt files</string>
```

Comment:
You shouldn't add a new release note, just modifying the one that was already there regarding the remove dialog

Reply:
deleted release note repeated